pages.fr/common/adb-forward.md

```diff
@@ -0,0 +1,20 @@
+# adb forward
+
+> Se connecte à un appareil Android sans fil.
+> Plus d'informations : <https://developer.android.com/tools/adb>.
+
+- Redirige un port TCP :
+
+`adb forward tcp:{{port_local}} tcp:{{port_distant}}`
+
```

Comment:
Add the missing example, https://github.com/tldr-pages/tldr/blob/main/pages/common/adb-forward.md

something like

```suggestion

- Redirige un port TCP vers un emulateur spécifique ou un appareil physique (par ID d'appareil / numéro de [s]érie) :
`adb -s {{ID_appareil}} forward tcp:{{port_local}} tcp:{{port_distant}}`
```